Misc/NEWS.d/next/Library/2020-05-30-08-10-23.bpo-40744.jKURVV.rst

```diff
@@ -0,0 +1,2 @@
+Drop support for SQLite3 pre v3.7.3. Patch by Sergey Fedoseev and Erlend E.
+Aasland.
```

Comment:
Could you add some extra explanation on *why* we are dropping support? (Basically some similar text as the one from the bpo)

Reply:
See commit 5d180f8. Maybe too much?

Reply:
Sorry, I should have been more explicit. I meant to add something referring to the fact that the module already cannot be built with sqlite3 pre 3.7.3. (So this is not a new restriction, just making it official).

Reply:
All right, thank you!